{"uid":"d55db15f15430193","name":"Переход на страницу с жалобами","fullName":"tests.test_excursium.TestUserCases#test_user_report_problem","historyId":"6ccf52d0675463b7f2ebb26dc5cf7f58","time":{"start":1753356093024,"stop":1753356103849,"duration":10825},"description":"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами","descriptionHtml":"<p>Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 969). Other element would receive the click: <a href=\"/documents/personal_data_processing_policy.pdf\" onclick=\"Helpers.ShowDocument('personal_data_processing_policy')\" class=\"mb-0\">...</a>\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x55a533dfc70e <unknown>\n#1 0x55a533898a51 <unknown>\n#2 0x55a5338e9701 <unknown>\n#3 0x55a5338e7cb9 <unknown>\n#4 0x55a5338e5851 <unknown>\n#5 0x55a5338e4fe3 <unknown>\n#6 0x55a5338d991a <unknown>\n#7 0x55a533906ff2 <unknown>\n#8 0x55a5338d90fd <unknown>\n#9 0x55a5339071be <unknown>\n#10 0x55a533926444 <unknown>\n#11 0x55a533906dc3 <unknown>\n#12 0x55a5338d7959 <unknown>\n#13 0x55a5338d84c1 <unknown>\n#14 0x55a533dc88aa <unknown>\n#15 0x55a533dcbc11 <unknown>\n#16 0x55a533dcb6a8 <unknown>\n#17 0x55a533dcc0b5 <unknown>\n#18 0x55a533db234c <unknown>\n#19 0x55a533dcc421 <unknown>\n#20 0x55a533d9ae2a <unknown>\n#21 0x55a533dea855 <unknown>\n#22 0x55a533deaa4e <unknown>\n#23 0x55a533dfb9f5 <unknown>\n#24 0x7f2d67b441c4 <unknown>","statusTrace":"self = <test_excursium.TestUserCases object at 0x7f4ea5c44510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2d314c75e48f02491bc4c69c3b9a1577\")>\n\n    @allure.title(\"Переход на страницу с жалобами\")\n    @allure.severity(\"Normal\")\n    @pytest.mark.user_case\n    def test_user_report_problem(self, driver):\n        \"\"\"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами\"\"\"\n        main_page = MainPage(driver)\n    \n        with allure.step(\"Открываем страницу\"):\n            driver.get(config.base_url)\n    \n        with allure.step(\"Переходим на страницу с экскурсиями\"):\n            driver.find_element(*main_page.all_excursion_btn).click()\n    \n        with allure.step(\"Находим и скролим до кнопки отправки жалобы\"):\n            report_btn = driver.find_element(*main_page.report_btn)\n            main_page.scroll_to(report_btn)\n>           report_btn.click()\n\ntests/test_excursium.py:110: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py:120: in click\n    self._execute(Command.CLICK_ELEMENT)\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py:573: in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4ea5bdca50>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...\\\\n#22 0x55a533deaa4e \\\\u003Cunknown>\\\\n#23 0x55a533dfb9f5 \\\\u003Cunknown>\\\\n#24 0x7f2d67b441c4 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 969). Other element would receive the click: <a href=\"/documents/personal_data_processing_policy.pdf\" onclick=\"Helpers.ShowDocument('personal_data_processing_policy')\" class=\"mb-0\">...</a>\nE         (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nE       Stacktrace:\nE       #0 0x55a533dfc70e <unknown>\nE       #1 0x55a533898a51 <unknown>\nE       #2 0x55a5338e9701 <unknown>\nE       #3 0x55a5338e7cb9 <unknown>\nE       #4 0x55a5338e5851 <unknown>\nE       #5 0x55a5338e4fe3 <unknown>\nE       #6 0x55a5338d991a <unknown>\nE       #7 0x55a533906ff2 <unknown>\nE       #8 0x55a5338d90fd <unknown>\nE       #9 0x55a5339071be <unknown>\nE       #10 0x55a533926444 <unknown>\nE       #11 0x55a533906dc3 <unknown>\nE       #12 0x55a5338d7959 <unknown>\nE       #13 0x55a5338d84c1 <unknown>\nE       #14 0x55a533dc88aa <unknown>\nE       #15 0x55a533dcbc11 <unknown>\nE       #16 0x55a533dcb6a8 <unknown>\nE       #17 0x55a533dcc0b5 <unknown>\nE       #18 0x55a533db234c <unknown>\nE       #19 0x55a533dcc421 <unknown>\nE       #20 0x55a533d9ae2a <unknown>\nE       #21 0x55a533dea855 <unknown>\nE       #22 0x55a533deaa4e <unknown>\nE       #23 0x55a533dfb9f5 <unknown>\nE       #24 0x7f2d67b441c4 <unknown>\n\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: ElementClickInterceptedException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1753356092694,"stop":1753356093023,"duration":329},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 969). Other element would receive the click: <a href=\"/documents/personal_data_processing_policy.pdf\" onclick=\"Helpers.ShowDocument('personal_data_processing_policy')\" class=\"mb-0\">...</a>\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x55a533dfc70e <unknown>\n#1 0x55a533898a51 <unknown>\n#2 0x55a5338e9701 <unknown>\n#3 0x55a5338e7cb9 <unknown>\n#4 0x55a5338e5851 <unknown>\n#5 0x55a5338e4fe3 <unknown>\n#6 0x55a5338d991a <unknown>\n#7 0x55a533906ff2 <unknown>\n#8 0x55a5338d90fd <unknown>\n#9 0x55a5339071be <unknown>\n#10 0x55a533926444 <unknown>\n#11 0x55a533906dc3 <unknown>\n#12 0x55a5338d7959 <unknown>\n#13 0x55a5338d84c1 <unknown>\n#14 0x55a533dc88aa <unknown>\n#15 0x55a533dcbc11 <unknown>\n#16 0x55a533dcb6a8 <unknown>\n#17 0x55a533dcc0b5 <unknown>\n#18 0x55a533db234c <unknown>\n#19 0x55a533dcc421 <unknown>\n#20 0x55a533d9ae2a <unknown>\n#21 0x55a533dea855 <unknown>\n#22 0x55a533deaa4e <unknown>\n#23 0x55a533dfb9f5 <unknown>\n#24 0x7f2d67b441c4 <unknown>","statusTrace":"self = <test_excursium.TestUserCases object at 0x7f4ea5c44510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2d314c75e48f02491bc4c69c3b9a1577\")>\n\n    @allure.title(\"Переход на страницу с жалобами\")\n    @allure.severity(\"Normal\")\n    @pytest.mark.user_case\n    def test_user_report_problem(self, driver):\n        \"\"\"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами\"\"\"\n        main_page = MainPage(driver)\n    \n        with allure.step(\"Открываем страницу\"):\n            driver.get(config.base_url)\n    \n        with allure.step(\"Переходим на страницу с экскурсиями\"):\n            driver.find_element(*main_page.all_excursion_btn).click()\n    \n        with allure.step(\"Находим и скролим до кнопки отправки жалобы\"):\n            report_btn = driver.find_element(*main_page.report_btn)\n            main_page.scroll_to(report_btn)\n>           report_btn.click()\n\ntests/test_excursium.py:110: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py:120: in click\n    self._execute(Command.CLICK_ELEMENT)\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py:573: in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4ea5bdca50>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...\\\\n#22 0x55a533deaa4e \\\\u003Cunknown>\\\\n#23 0x55a533dfb9f5 \\\\u003Cunknown>\\\\n#24 0x7f2d67b441c4 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 969). Other element would receive the click: <a href=\"/documents/personal_data_processing_policy.pdf\" onclick=\"Helpers.ShowDocument('personal_data_processing_policy')\" class=\"mb-0\">...</a>\nE         (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nE       Stacktrace:\nE       #0 0x55a533dfc70e <unknown>\nE       #1 0x55a533898a51 <unknown>\nE       #2 0x55a5338e9701 <unknown>\nE       #3 0x55a5338e7cb9 <unknown>\nE       #4 0x55a5338e5851 <unknown>\nE       #5 0x55a5338e4fe3 <unknown>\nE       #6 0x55a5338d991a <unknown>\nE       #7 0x55a533906ff2 <unknown>\nE       #8 0x55a5338d90fd <unknown>\nE       #9 0x55a5339071be <unknown>\nE       #10 0x55a533926444 <unknown>\nE       #11 0x55a533906dc3 <unknown>\nE       #12 0x55a5338d7959 <unknown>\nE       #13 0x55a5338d84c1 <unknown>\nE       #14 0x55a533dc88aa <unknown>\nE       #15 0x55a533dcbc11 <unknown>\nE       #16 0x55a533dcb6a8 <unknown>\nE       #17 0x55a533dcc0b5 <unknown>\nE       #18 0x55a533db234c <unknown>\nE       #19 0x55a533dcc421 <unknown>\nE       #20 0x55a533d9ae2a <unknown>\nE       #21 0x55a533dea855 <unknown>\nE       #22 0x55a533deaa4e <unknown>\nE       #23 0x55a533dfb9f5 <unknown>\nE       #24 0x7f2d67b441c4 <unknown>\n\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: ElementClickInterceptedException","steps":[{"name":"Открываем страницу","time":{"start":1753356093024,"stop":1753356094566,"duration":1542},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Переходим на страницу с экскурсиями","time":{"start":1753356094566,"stop":1753356102704,"duration":8138},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Находим и скролим до кнопки отправки жалобы","time":{"start":1753356102704,"stop":1753356103848,"duration":1144},"status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 969). Other element would receive the click: <a href=\"/documents/personal_data_processing_policy.pdf\" onclick=\"Helpers.ShowDocument('personal_data_processing_policy')\" class=\"mb-0\">...</a>\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x55a533dfc70e <unknown>\n#1 0x55a533898a51 <unknown>\n#2 0x55a5338e9701 <unknown>\n#3 0x55a5338e7cb9 <unknown>\n#4 0x55a5338e5851 <unknown>\n#5 0x55a5338e4fe3 <unknown>\n#6 0x55a5338d991a <unknown>\n#7 0x55a533906ff2 <unknown>\n#8 0x55a5338d90fd <unknown>\n#9 0x55a5339071be <unknown>\n#10 0x55a533926444 <unknown>\n#11 0x55a533906dc3 <unknown>\n#12 0x55a5338d7959 <unknown>\n#13 0x55a5338d84c1 <unknown>\n#14 0x55a533dc88aa <unknown>\n#15 0x55a533dcbc11 <unknown>\n#16 0x55a533dcb6a8 <unknown>\n#17 0x55a533dcc0b5 <unknown>\n#18 0x55a533db234c <unknown>\n#19 0x55a533dcc421 <unknown>\n#20 0x55a533d9ae2a <unknown>\n#21 0x55a533dea855 <unknown>\n#22 0x55a533deaa4e <unknown>\n#23 0x55a533dfb9f5 <unknown>\n#24 0x7f2d67b441c4 <unknown>\n\n","statusTrace":"  File \"/app/tests/test_excursium.py\", line 110, in test_user_report_problem\n    report_btn.click()\n    ~~~~~~~~~~~~~~~~^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py\", line 120, in click\n    self._execute(Command.CLICK_ELEMENT)\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webelement.py\", line 573, in _execute\n    return self._parent.execute(command, params)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Скроллим до элемента","time":{"start":1753356102715,"stop":1753356102776,"duration":61},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"8e3dc90a895034fa","name":"Скриншот при падении","source":"8e3dc90a895034fa.png","type":"image/png","size":408934}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1753356104040,"stop":1753356104103,"duration":63},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Пользовательские сценарии"},{"name":"severity","value":"Normal"},{"name":"tag","value":"user_case"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_excursium"},{"name":"subSuite","value":"TestUserCases"},{"name":"host","value":"bcb52ef2d38b"},{"name":"thread","value":"19-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_excursium"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["user_case"]},"source":"d55db15f15430193.json","parameterValues":[]}